test/e2e/test_cluster_configs/dualstack.json

```diff
@@ -0,0 +1,54 @@
+{
+    "env": {},
+    "apiModel": {
```

Comment:
Let's add this between `"env"` and `"apiModel"`:

```
"options": {
		"allowedOrchestratorVersions": ["latestReleasedVersion"]
	},
```

This will make sure it tests 1.19-only, which seems like a good compromise given that it doesn't support 1.16 & 1.17.

Reply:
Done!